unity-renderer/ProjectSettings/ProjectSettings.asset

```diff
@@ -17,7 +17,7 @@ PlayerSettings:
   defaultCursor: {fileID: 0}
   cursorHotspot: {x: 0, y: 0}
   m_SplashScreenBackgroundColor: {r: 1, g: 1, b: 1, a: 1}
-  m_ShowUnitySplashScreen: 0
+  m_ShowUnitySplashScreen: 1
```

Comment:
```suggestion
  m_ShowUnitySplashScreen: 0
```

Can you revert this file?

Reply:
reverted ProjectSettings.asset file as it was before